### ISSUE TITLE
TypeError: 'Struct' object is not iterable

### ISSUE THREAD

**Issue by sriramkg86** [chuckorde/veracode-python#8]
Traceback (most recent call last):
  File "Veracode_Gate_Script.py", line 42, in <module>
    v_results = v.get_scan_results(application_name, sandbox_name, version_id)
  File "F:\CIO_Metrics\Veracode_API.py", line 15, in get_scan_results
    app = Application(app_name)
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\veracode\application.py", line 22, in __new__
    return ExistingApplication(name, sandbox, build)
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\veracode\application.py", line 146, in __init__
    self.info = self._get_app_by_name(app_name).application
  File "C:\Program Files (x86)\Python38-32\lib\site-packages\veracode\application.py", line 180, in _get_app_by_name
    for app in apps.app:
TypeError: 'Struct' object is not iterable
##[error]Cmd.exe exited with code '1'.

**Comment by sriramkg86** [comment]
at run time it results as an array list, I think this is the root cause where it causing an exception.                                                                                                                                               
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<applist xmlns="[https://analysiscenter.veracode.com/schema/2.0/applist"](https://analysiscenter.veracode.com/schema/2.0/applist%22) xmlns:xsi="[http://www.w3.org/2001/XMLSchema-instance"](http://www.w3.org/2001/XMLSchema-instance%22) account_id="11810" applist_version="1.2" xsi:schemaLocation="https://analysiscenter.veracode.com/schema/2.0/applist [https://analysiscenter.veracode.com/resource/2.0/applist.xsd">](https://analysiscenter.veracode.com/resource/2.0/applist.xsd%22%3E)
  <app app_id="1284269" app_name="AM Toolkit" policy_updated_date="2022-02-25T16:05:38-05:00"/>
  <app app_id="1284265" app_name="AutoRouter" policy_updated_date="2022-02-24T13:21:43-05:00"/>
  <app app_id="1284283" app_name="Billing Engine" policy_updated_date="2022-02-04T12:16:14-05:00"/>
  <app app_id="765050" app_name="CardPro Connect" policy_updated_date="2022-02-22T14:08:18-05:00"/>
  <app app_id="1283870" app_name="CIDMS"/>
  <app app_id="765061" app_name="CIIS" policy_updated_date="2021-12-10T11:00:38-05:00"/>
  <app app_id="1284274" app_name="Coupa Service" policy_updated_date="2022-02-25T15:37:26-05:00"/>
  <app app_id="765072" app_name="DPE" policy_updated_date="2022-02-25T00:57:30-05:00"/>
  <app app_id="1284249" app_name="Embossing Scanners - Software (Scan App)"/>
  <app app_id="1103683" app_name="FIS Chip Card Plug-In"/>
  <app app_id="1284262" app_name="InspectView - RV"/>
  <app app_id="1303079" app_name="Inspectview - SP"/>
  <app app_id="1284252" app_name="Inventory" policy_updated_date="2022-02-28T02:50:22-05:00"/>
  <app app_id="1283851" app_name="Inventory Management System (IMS)"/>
  <app app_id="1284478" app_name="Issuer Import Tool"/>
  <app app_id="1284267" app_name="Job Mover" policy_updated_date="2022-02-09T08:51:57-05:00"/>
  <app app_id="771100" app_name="Job Tracker" policy_updated_date="2022-02-16T07:52:07-05:00"/>
  <app app_id="1284273" app_name="Job Tracker Reports" policy_updated_date="2022-02-18T03:10:58-05:00"/>
  <app app_id="1284278" app_name="Router Editor"/>
  <app app_id="1284258" app_name="Transit System App"/>
  <app app_id="1284487" app_name="ViewNew"/>
</applist> 

Basically, need to put a filter at the function definition " _get_app_by_name" to  select the appropriate entry to return if the input has an array list of applications.
